recipes/grz-cli/meta.yaml

```
@@ -1,13 +1,13 @@
 {% set name = "grz-cli" %}
-{% set version = "0.1.1" %}
+{% set version = "0.1.4" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/grz_cli-{{ version }}.tar.gz
-  sha256: 199dd5d96405065db6224d75e991dd385b28685bc7f581e75a8c09d913737fa7
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/grz_cli-{{ version }}.tar.gz
```

```suggestion
  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/grz_cli-{{ version }}.tar.gz
```
Is the `.io` intentional here? Because I think it should be `.org`, based on these discussions:
* https://github.com/pypi/warehouse/issues/2184

---

Yes, intentional.
It seems like the autobump machinery did not pick up on the .org URL, not sure why.

---

Maybe an old grayskull version or something?
https://github.com/conda/grayskull/issues/537

---

I mean, it was .org before, but for that autobump did not recognize it as a known source url (for pypi), so perhaps there is some issue in the autobump source detection. I'll change it back to .org here and see if I can find the underlying issue.